src/vm/callingconvention.h

```diff
@@ -326,9 +330,13 @@ class ArgIteratorTemplate : public ARGITERATOR_BASE
     UINT GetFPReturnSize()
     {
         WRAPPER_NO_CONTRACT;
+#ifdef ARM_SOFTFP
+        return 0;
```

Comment:
This should rather be fixed in the `ArgIteratorTemplate<ARGITERATOR_BASE>::ComputeReturnFlags()` by excluding the two cases for ELEMENT_TYPE_R4 and ELEMENT_TYPE_R8 for ARM_SOFTFP.